subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -3245,6 +3323,17 @@ SHELL_SUBCMD_ADD((wifi), wps_pin, &wifi_commands,
 		 "[pin] Only applicable for set.\n",
 		 cmd_wifi_wps_pin,
 		 1, 1);
+SHELL_SUBCMD_ADD((wifi), 11k, &wifi_commands,
+		 "Configure 11k or get 11k status.\n"
+		 "[enable/disable]\n",
+		 cmd_wifi_11k,
+		 1, 1);
+
+SHELL_SUBCMD_ADD((wifi), 11k_neighbor_request, &wifi_commands,
+		 "Send Neighbor Report Request frame.\n"
+		 "[ssid <ssid>]\n",
+		 cmd_wifi_11k_neighbor_request,
+		 1, 2);
```

Comment:
Could you place these in alphabetical order in the command list (so at top), it is easier to find the commands that way.

Reply:
Updated.